Lib/test/test_zipfile/_path/test_path.py

```diff
@@ -454,9 +454,15 @@ def test_root_unnamed(self, alpharep):
 
         # .name and .parent should still work on subs
         sub = root / "b"
-        assert sub.name == "b"
+        self.assertEqual(sub.name, "b")
```

Comment:
This project uses bare asserts because it's [backported to zipp](https://github.com/python/importlib_metadata/wiki/Development-Methodology). The bare asserts work fine in CPython and provide better fidelity and attribution when running with assert rewrites. Let's stick to bare asserts.